packages/docusaurus-theme-translations/locales/uk/theme-common.json

```diff
@@ -9,7 +9,7 @@
   "theme.DocSidebarItem.expandCategoryAriaLabel": "Expand sidebar category '{label}'",
   "theme.ErrorPageContent.title": "На сторінці стався збій.",
   "theme.ErrorPageContent.tryAgain": "Спробуйте ще раз",
-  "theme.NavBar.navAriaLabel": "Main",
+  "theme.NavBar.navAriaLabel": "Головна",
```

Comment:
3 lines above, `theme.DocSidebarItem.expandCategoryAriaLabel` is not translated, can you translate it too?